src/solvers/refinement/string_refinement.cpp

```diff
@@ -187,7 +187,8 @@ void string_refinementt::add_symbol_to_symbol_map(
   symbol_resolve[lhs]=new_rhs;
   reverse_symbol_resolve[new_rhs].push_back(lhs);
 
-  std::list<exprt> symbols_to_update_with_new_rhs(reverse_symbol_resolve[rhs]);
+  const std::list<exprt>
```

Comment:
That looks  a bit weird. I'd rather break after the ( or use assignment notation and break after =.